<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainW</class>
 <widget class="QMainWindow" name="MainW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainW</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>25</number>
      </property>
      <item>
       <widget class="QListView" name="fileListView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="addFileBtn">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeFileBtn">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeAllBtn">
          <property name="text">
           <string>删除所有</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>合成文件名:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outFileNameEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="compositeBtn">
        <property name="text">
         <string>合成</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="readOnlyCheckBox">
        <property name="text">
         <string>只读</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>禁止打印</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>允许打印</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="lockedCheckBox">
        <property name="text">
         <string>加密</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwdEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showPasswd">
        <property name="text">
         <string>明文</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLocked"/>
   <addaction name="actionUnlocked"/>
   <addaction name="actionSplitFile"/>
   <addaction name="actionSeting"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionLocked">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons8-lock-100.png</normaloff>:/images/icons8-lock-100.png</iconset>
   </property>
   <property name="text">
    <string>Locked</string>
   </property>
   <property name="toolTip">
    <string>加密文件</string>
   </property>
  </action>
  <action name="actionUnlocked">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons8-padlock-100.png</normaloff>:/images/icons8-padlock-100.png</iconset>
   </property>
   <property name="text">
    <string>unlocked</string>
   </property>
   <property name="toolTip">
    <string>解密文件</string>
   </property>
  </action>
  <action name="actionSplitFile">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons8-front-sorting-96.png</normaloff>:/images/icons8-front-sorting-96.png</iconset>
   </property>
   <property name="text">
    <string>SplitFile</string>
   </property>
   <property name="toolTip">
    <string>拆分文件</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons8-help-96.png</normaloff>:/images/icons8-help-96.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>版权信息</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons8-about-96.png</normaloff>:/images/icons8-about-96.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>关于软件</string>
   </property>
  </action>
  <action name="actionSeting">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons8-settings-96.png</normaloff>:/images/icons8-settings-96.png</iconset>
   </property>
   <property name="text">
    <string>Seting</string>
   </property>
   <property name="toolTip">
    <string>设置存储位置</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
